tests/test_record_values.py

```diff
@@ -31,6 +31,9 @@
     "might think to put into a record that can theoretically hold a huge " \
     "string and so lets test it and prove that shall we?"
 
+# The numpy dtype of all arrays of strings
+NUMPY_DTYPE_STRING = "S40"
```

Comment:
Don't think you use this anymore

Reply:
Removed in 5b59ed7f86a126842467247dfb004ecf21aed561